tests/unit/dag/test_executors.py

```diff
@@ -0,0 +1,104 @@
+#
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+import pandas as pd
+
+from merlin.core.dispatch import make_df
+from merlin.dag import DictArray, Graph
+from merlin.dag.base_operator import BaseOperator
+from merlin.dag.executors import LocalExecutor
+from merlin.schema.schema import ColumnSchema, Schema
+
+
```

Comment:
The tests in this file demonstrate the capabilities we discovered were needed in order to process the kinds of data we deal with after loading data from disk and before it gets passed to models, like operating on dictionary-like objects and transforming tuples of data (e.g. `(X, y)`) with the same transformation graph.